srv/components/misc_pkgs/kibana/files/kibana.yml.j2

```diff
@@ -43,7 +43,7 @@
 # the username and password that the Kibana server uses to perform maintenance on the Kibana
 # index at startup. Your Kibana users still need to authenticate with Elasticsearch, which
 # is proxied through the Kibana server.
-#elasticsearch.username: "kibana"
+#elasticsearch.username: "user"
 #elasticsearch.password: "pass"
```

Comment:
Better remove this password before there's another ticket raised :)

Reply:
@ypise  I believe next time we should refuse such tickets.